### PR TITLE
ch. 4 end region tags

### PR DESCRIPTION
**Pull request by HannaMichaelis**

### CODE REVIEW

**Comment by HannaMichaelis** [comment]
No clue why these tests aren't passing

**Comment by BenjaminMichaelis** [comment]
@HannaMichaelis 
In 4.29, looks like some code was deleted before the tag added.

image

File: src/Chapter04/Listing04.06.CharDifferences.cs [hunk 1]
```diff
@@ -6,8 +6,7 @@ public static void Main()
         {
             #region INCLUDE
             int distance = 'f' - 'c';
-            System.Console.WriteLine(distance);
```
Comment (Keboo) [inline src/Chapter04/Listing04.06.CharDifferences.cs:9]: Looks like this line got removed?

File: src/Chapter04/Listing04.39.RightShiftOperator.cs [hunk 2]
```diff
@@ -8,9 +8,8 @@ public static void Main()
             int x;
             x = (-7 >> 2); // 11111111111111111111111111111001 becomes 
                            // 11111111111111111111111111111110
-            // Write out "x is -2."
-            System.Console.WriteLine($"x = { x }.");
```
Comment (Keboo) [inline src/Chapter04/Listing04.39.RightShiftOperator.cs]: Looks like this code got delete

**Review by Keboo — CHANGES_REQUESTED** [review]

File: src/Chapter04/Listing04.39.RightShiftOperator.cs [hunk 3]
```diff
@@ -8,9 +8,9 @@ public static void Main()
...
-            #endregion
+                           // Write out "x is -2."
+            System.Console.WriteLine($"x = {x}.");
```
Comment (BenjaminMichaelis) [inline src/Chapter04/Listing04.39.RightShiftOperator.cs]:
```suggestion
            // Write out "x is -2."
            System.Console.WriteLine($"x = {x}.");
```

**Review by BenjaminMichaelis — APPROVED** [review]

**Pull request merged**